app-interface/manifests/rhose-flee-manager.yml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: rhose-flee-manager
+  labels:
+    app: rhose-flee-manager
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: fleet-manager-bridge
+      labels:
+        app-component: fleet-manager
+    spec:
+      selector:
+        matchLabels:
+          app-component: fleet-manager
+      replicas: ${{FLEET_MANAGER_DEPLOYMENT_REPLICAS}}
+      template:
+        metadata:
+          labels:
+            app-component: fleet-manager
+        spec:
+          containers:
+            - image: ${{FLEET_MANAGER_IMAGE}}:${{{FLEET_MANAGER_IMAGE_TAG}}}
+              imagePullPolicy: IfNotPresent
+              name: rhose-fleet-manager
+              resources:
+                limits:
+                  cpu: ${{FLEET_MANAGER_CPU_LIMITS}}
+                  memory: ${{FLEET_MANAGER_MEM_LIMITS}}
+                requests:
+                  cpu: ${{FLEET_MANAGER_CPU_REQS}}
+                  memory: ${{FLEET_MANAGER_MEM_REQS}}
+              ports:
+                - containerPort: ${{FLEET_MANAGER_HTTP_TRANSPORT_PORT}}
+              envFrom:
+                - configMapRef:
+                    name: fleet-manager-bridge-config
+                - secretRef:
+                    name: fleet-manager-bridge-secrets
+              livenessProbe:
+                httpGet:
+                  path: ${{FLEET_MANAGER_LIVENESS_PATH}}
```

Comment:
I don't think we'll ever need to the LIVENESS/READINESS path/port, so I'd suggest hardcoding it just to minimise the number of parameters on the template.